src/pudl/transform/eia861.py

```diff
@@ -818,5 +853,9 @@ def transform(raw_dfs, eia861_tables=pc.pudl_tables["eia861"]):
             logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
                         f"concatenated across all years.")
             eia861_transform_functions[table](raw_dfs, tfr_dfs)
-
+    # for each transformed table, convert year col to date
+    for table in tfr_dfs:
+        tfr_dfs[table] = (
+            tfr_dfs[table].pipe(pudl.helpers.convert_to_date)
+        )
```

Comment:
This ought to be redundant now, right? Since the date conversions are being done beforehand?

Reply:
agreed